toolkits/global/packages/global-facet/README.md

```diff
@@ -0,0 +1,87 @@
+# Global Facet
+
+Global branded facet buttons.
+
+## CSS
+
+Include the SCSS in your application
+
+```scss
+// Pick ONE of the brands below to include
+@import '@springernature/global-facet/scss/10-settings/default';
+@import '@springernature/global-facet/scss/10-settings/nature';
+
+// Include this with your other components
+@import '@springernature/global-facet/scss/50-components/facet';
+@import '@springernature/global-facet/scss/50-components/facet-expander';
+```
+
+## Example usage
+
+```html
+<div class="c-facet" data-facet>
+    <!-- example of checkbox input -->
+    <div class="c-facet__container">
+        <fieldset class="c-facet__item">
+            <legend>
+                <span class="c-facet__label">Subject</span>
+                <span class="u-hide u-js-show">
+                    <button type="button" class="c-facet__button c-facet__button--border" data-facet-expander data-facet-target="#subject-target">
+                        <svg class="c-facet__icon" role="img" aria-hidden="true" focusable="false" height="16"
```

Comment:
I think we can remove the width and height attributes from the example <svg> here, as per the outcomes of the discussions Heydon Pickering started the other day. https://springernature.slack.com/archives/C0GDUEGMS/p1631007607070800

Reply:
Oh scratch that - I've just seen that you are including the `u-icon` mixin within the facet scss 👍